cluster/terraform_kubernetes/welcome_app.tf

```diff
@@ -114,3 +167,29 @@ resource "kubernetes_ingress_v1" "welcome_app" {
     }
   }
 }
+
+resource "kubernetes_config_map" "nginx_config" {
```

Comment:
Can we rename to welcome_app_nginx_config ?

Reply:
fixed